src/mngt/ocf_mngt_cache.c

```diff
@@ -1705,6 +1708,7 @@ static void ocf_mngt_cache_stop_put_io_queues(ocf_pipeline_t pipeline,
 	ocf_cache_t cache = context->cache;
 
 	_ocf_mngt_cache_put_io_queues(cache);
+	env_spinlock_destroy(&cache->io_queues_list_lock);
```

Comment:
Shouldn't this be in ocf_mngt_cache_remove instead?

Reply:
Since the lock is related to the list of queues they are initialized one after another, the deinitialization might be done is the same manner